presto-hive/src/test/java/io/prestosql/plugin/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -508,7 +535,32 @@ public void testSchemaAuthorizationForRole()
                 .setIdentity(Identity.forUser("user").withPrincipal(getSession().getIdentity().getPrincipal()).build())
                 .build();
 
+        Session anotherUser = testSessionBuilder()
```

Comment:
Please have dedicated tests for `CREATE SCHEMA .. AUTHORIZATION ROLE ...`.
If you mix this with ALTER SCHEMA it is getting very complicated.